### PR TITLE
chore: add link and Python 3.8/3.9 classifiers to setup.py

### PR DESCRIPTION
**Pull request by jli**
The link should appear on PyPI in the future: https://pypi.org/project/kfp/#description
The read-the-docs page has further links to the Github repo and other documentation too, so seemed like a decent choice.

### CODE REVIEW

**Comment by jli** [comment]
@googlebot I signed it!

File: sdk/python/setup.py [hunk 1]
```diff
@@ -77,7 +77,8 @@ def find_version(*file_path_parts):
     name=NAME,
     version=find_version('kfp', '__init__.py'),
     description='KubeFlow Pipelines SDK',
-    author='google',
+    author='Google',
```
Comment (NikeNano) [inline sdk/python/setup.py:80]: Should we update the author here to `kubeflow authors` similar to https://github.com/kubeflow/pipelines/issues/5617

Comment (jli) [reply sdk/python/setup.py]: done!

File: sdk/python/setup.py [hunk 2]
```diff
@@ -77,7 +77,8 @@ def find_version(*file_path_parts):
     name=NAME,
     version=find_version('kfp', '__init__.py'),
     description='KubeFlow Pipelines SDK',
-    author='google',
+    author='The Kubeflow Authors',
+    url='https://kubeflow-pipelines.readthedocs.io/en/stable/',
```
Comment (Ark-kun) [inline sdk/python/setup.py:81]: I wonder whether we should link to the repo.

Comment (Ark-kun) [reply sdk/python/setup.py:81]:
```
    url="https://github.com/kubeflow/pipelines",
    project_urls={
        "Documentation": "https://kubeflow-pipelines.readthedocs.io/en/stable/",
        "Bug Tracker": "https://github.com/kubeflow/pipelines/issues",
    },
```

Comment (jli) [reply sdk/python/setup.py]: done!

**Comment by jli** [comment]
I fixed merge conflicts and added more URLs per @Ark-kun 's suggestion. PTAL :)

**Pull request merged**